src/js/actions/documents.js

```diff
@@ -856,12 +860,119 @@ define(function (require, exports) {
     beforeStartup.writes = [locks.JS_DOC];
 
     /**
-     * Initialize the inactive documents. (The active document is initialized beforeStartup.)
+     * Send info to search store about searching for documents and
+     * initialize the inactive documents. (The active document is initialized beforeStartup.)
      *
      * @param {{currentIndex: number, docCount: number}=} payload
      * @return {Promise}
      */
     var afterStartup = function (payload) {
+        var appStore = this.flux.store("application"),
+            docStore = this.flux.store("document");
+
+        /**
+         * Make list of currently open documents to be used as search options
+         * 
+         * @return {Immutable.List.<object>}
+        */
+        var currDocSearchOptions = function () {
+            var document = appStore.getCurrentDocument(),
+                openDocs = appStore.getOpenDocumentIDs().filterNot(function (doc) {
+                                return doc === document.id;
+                            }),
+                docMap = openDocs.map(function (doc) {
+                    return {
+                        id: "curr-doc_" + doc.toString(),
+                        title: docStore.getDocument(doc).name,
+                        type: "item",
+                        category: [strings.SEARCH.CATEGORIES.DOCUMENT, strings.SEARCH.CATEGORIES.CURRENT]
+                    };
+                }),
+                docLabel = {
```

Comment:
Are you suggesting that some of the information (like the ID type) would be set for the option in the search store while other information (like the name) would be set here in the document action?